client/src/components/Overview/Cart.jsx

```diff
@@ -25,7 +25,7 @@ const Cart = ({selectedStyle, skuChange, selectedSizeQuantity, selectedSku, sele
      skuAlertChange(true);
      console.log(skuSelectRef.current);
      // TODO: not working
-     skuSelectRef.current.click();
+     skuSelectRef.current.focus();
```

Comment:
What is the current.focus() functionality for/what is it intended to light up/focus on? 

To select the current sku reference element? (show the client the next available selection)

Reply:
it will make the select focused, shown with a blue box surround it in the UI. As per the doc, 
'If the default ‘Select Size’ is currently selected: Clicking this button should open the size dropdown, and a message should appear above the dropdown stating “Please select size”.' 
I tried to use current.click to open the select, but it's not working, so I just use .focus() as a workaround